### DISCUSSION TITLE
Welcome to pykt-toolkit Discussions!

### DISCUSSION THREAD

**Discussion by pykt-team** [pykt-team/pykt-toolkit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by linchuanghong** [comment]
assistment2017这个数据集中 (anonymized_full_release_competition_dataset.csv) 没有看到concepts这个特征的数据，但是经过预处理之后（data_preprocess.py）之后，生成的数据文件里，包括了concepts这个特征数据？请问是怎么得到的？
